src/rpc/blockchain.cpp

```diff
@@ -515,9 +515,10 @@ static RPCHelpMan getblockheader()
                 {
                     {"blockhash", RPCArg::Type::STR_HEX, RPCArg::Optional::NO, "The block hash"},
                     {"verbose", RPCArg::Type::BOOL, RPCArg::Default{true}, "true for a json object, false for the hex-encoded data"},
+                    {"count", RPCArg::Type::AMOUNT, RPCArg::Default{"null"}, "Number of headers to get (maximum: 2000)"},
```

Comment:
I don't think this needs to be an AMOUNT, NUM would be preferred I think? We don't want floats or strings here, just integers.
```suggestion
                    {"count", RPCArg::Type::NUM, RPCArg::Optional::OMITTED, "Number of headers to get (maximum: 2000)"},
```